pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-osd/default.nix

```diff
@@ -2,21 +2,19 @@
 , autoreconfHook } :
 
 stdenv.mkDerivation rec {
-  name = "pidgin-osd-0.1.0";
+  name = "pidgin-osd-0.2.0";
   src = fetchurl {
-    url = https://github.com/mbroemme/pidgin-osd/archive/pidgin-osd-0.1.0.tar.gz;
-    sha256 = "11hqfifhxa9gijbnp9kq85k37hvr36spdd79cj9bkkvw4kyrdp3j";
+    url = https://github.com/edanaher/pidgin-osd/archive/pidgin-osd-0.2.0.tar.gz;
+    sha256 = "1dfb0957wwm5zly4w4g815svhkhvjbj3dgik1lbyac8adh9ks784";
```

Comment:
I'd be fine with `-unstable-date`, though given that the last commit before my changes was in 2013, and I would be surprised if there's another commit this year, so it might be a bit misleading.

But I don't follow your request to "change the version number the same" - they're both 0.2.0, and I don't see what would irritate users.  Am I missing something?